sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+SRC_URI="https://github.com/sigma-star/mtd-utils/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+S="${WORKDIR}/${PN}-v${PV}-0ef86b1"
+
+DESCRIPTION="MTD userspace tools (NFTL, JFFS2, NAND, FTL, UBI)"
+HOMEPAGE="http://git.infradead.org/?p=mtd-utils.git;a=summary"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+lzo xattr +zstd"
+
+# We need libuuid
+RDEPEND="!sys-fs/mtd
+	>=sys-apps/util-linux-2.16
+	sys-libs/zlib
+	lzo? ( dev-libs/lzo )
+	zstd? ( app-arch/zstd )"
+# ACL is only required for the <sys/acl.h> header file to build mkfs.jffs2
+# And ACL brings in Attr as well.
+DEPEND="${RDEPEND}
+	xattr? ( sys-apps/acl )"
+BDEPEND="sys-devel/libtool"
+
+src_prepare() {
+	default
+	./autogen.sh || die
+}
+
+src_configure() {
+	econf \
+		$(use_with lzo) \
+		$(use_with xattr) \
+		$(use_with zstd)
+}
+
+src_install() {
+	default
+	dodoc jffsX-utils/device_table.txt
+	newdoc ubifs-utils/mkfs.ubifs/README README.mkfs.ubifs
```

Comment:
You could add a `DOCS` array before `src_prepare` so these'd get installed with the `default` command here.

Reply:
Done, thanks